packages/ui/src/components/shadcn/ui/menubar.tsx

```diff
@@ -93,7 +93,7 @@ const MenubarContent = React.forwardRef<
       alignOffset={alignOffset}
       sideOffset={sideOffset}
       className={cn(
-        'z-50 min-w-[12rem] overflow-hidden rounded-md border bg-overlay p-1 text-popover-foreground shadow-md animate-in slide-in-from-top-1',
```

Comment:
this will break animation on www

Reply:
think you might be using wrong component, this uses role="menubar".
why not use navigation-menu ?

Reply:
I was using the navigation-menu at first, but it wasn't working well with the overflow-x-scroll I needed on mobile. The overflow was cropping the dropdown.

Reply:
but yeah probably need to do this via className prop rather than the global component itself...

Reply:
I can update navigation-menu to work better. 
We can probably wrap the NavigationMenuList in a Scrollable component and should all work.

Reply:
this menubar change wasn't needed after all btw, reverted it 👍

Reply:
Oh, now I remember why I removed those classes. It was to fix [this](https://github.com/shadcn-ui/ui/issues/25) issue. @MildTomato 

https://github.com/supabase/supabase/assets/25671831/c71e2506-85b0-46d1-a818-8726664c9897

Reply:
https://github.com/agmmnn/tauri-ui/commit/0faa80dcf6a4cd6c8dd005cc591bb9600f232d2b

looks like we already removed these classes

Reply:
ahh, that recording is of the MenuBar version ?

Reply:
yeah the one I previously made the change on this PR. Restoring it has this issue again (will update to try to use the navigation-menu tho)

Reply:
this is solved now?

Reply:
I reverted these changes from this PR since I'm not using menubar anymore. That freezing issue still subsists.

Can fix it in a separate PR.